src/test/rgw/rgw_multi/tests_es.py

```diff
@@ -226,16 +227,16 @@ def create_obj(k, i):
                 do_check_mdsearch(target_conn.conn, bucket, src_keys , 'x-amz-meta-foo-date > ' + sval, lambda k: date_from_str(k.get_metadata('foo-date')) > val)
 
             # 'or' query
-            for i in xrange(len(src_keys) / 2):
+            for i in range(len(src_keys) / 2):
```

Comment:
I guess you need to use `//` instead of `/`.

Reply:
Just clarify:
```
Python 2.7.17 (default, Dec 23 2019, 21:25:33) 
[GCC 4.2.1 Compatible Apple LLVM 11.0.0 (clang-1100.0.33.16)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> 3/2
1
>>> 3//2
1
```
and
```
Python 3.7.3 (default, Mar  6 2020, 22:34:30) 
[Clang 11.0.3 (clang-1103.0.32.29)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> 3/2
1.5
>>> 3//2
1
```

Reply:
thanks fixed.